sklearn/feature_selection/_rfe.py

```diff
@@ -544,6 +545,11 @@ def fit(self, X, y, groups=None):
             train/test set. Only used in conjunction with a "Group" :term:`cv`
             instance (e.g., :class:`~sklearn.model_selection.GroupKFold`).
 
+        error_score : 'raise' or numeric, default=np.nan
+            Value to assign to the score if an error occurs in scoring
+            fitting. If set to 'raise', the error is raised.
+            If a numeric value is given, UserWarning is raised.
```

Comment:
Adding this feature is outside the scope of the PR. For the call to `_score` in `RFE`, lets set `error_score='raise'` which is the behavior on master.